docs/howtos/layers/image.md

```diff
@@ -397,17 +398,48 @@ value.
 
 As of right now, adjusting the contrast limits has no effect for `rgb` data.
 
+If you pass contrast limits as a keyword argument to a layer, then the full
+extent of the `contrast limits:` range slider will be set to those values.
```

Comment:
````suggestion
```{important}
If you pass `contrast_limits` as a keyword argument to a layer, then the full
extent of the `contrast limits:` range slider will also be set to those values!
```
````
This is really insidious, lets make it an admonition.